extensions/mssql/src/sqlClusterLookUp.ts

```diff
@@ -0,0 +1,147 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import * as sqlops from 'sqlops';
+import * as constants from './constants';
+import * as UUID from 'vscode-languageclient/lib/utils/uuid';
+import { AppContext } from './appContext';
+import { SqlClusterConnection } from './objectExplorerNodeProvider/connection';
+import { ICommandObjectExplorerContext, ICommandUnknownContext, ICommandViewContext } from './objectExplorerNodeProvider/command';
+import { SqlObjectExplorerNodeProvider } from './objectExplorerNodeProvider/objectExplorerNodeProvider';
+
+export class SqlClusterLookUp {
+	public static findSqlClusterConnection(
```

Comment:
As mentioned offline, this kind of function should just be a direct `export function findSqlClusterConnection` in Javascript. No need to put inside a class as it has overhead. Please convert this to a set of public `export function` calls with private functions.

Reply:
fixed.